### PR TITLE
fix #303403: play repeats button doesn't reflect stored settings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Fixes https://musescore.org/en/node/303403 (and came up in https://musescore.org/en/node/303316#comment-989518)

While at it fix the same bug for the pan button too (which just came up in https://musescore.org/en/node/303874#comment-992514)

Not sure why I didn't notice those 2 bugs and fixed along with #5378

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -445,7 +445,7 @@ void MuseScore::preferencesChanged(bool fromWorkspace)
       {
       updateExternalValuesFromPreferences();
 
-      setPlayRepeats(MScore::playRepeats);
```
Comment (anatoly-os) [inline mscore/musescore.cpp:448]: Why did you decide to remove the call of all other operations included to this method, and left only the checked state setter?
Isn't the call of `setPlayRepeats` necessary here?

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp:448]: It doesn't seem to be as per my testing

Comment (anatoly-os) [reply mscore/musescore.cpp:448]: @Soerboe added this code for some reason... https://github.com/musescore/MuseScore/commit/064156c267c6b3b576ac133f064644a9172f300e

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp:448]: Maybe, but it didn't work, not for showing the current real status

**Pull request merged**